neo/utils/datasets.py

```diff
@@ -0,0 +1,83 @@
+"""
+Some simple function to retrieve public datasets.
+"""
+from pathlib import Path
+import os
+
+try:
+    import datalad.api
+    HAVE_DATALAD = True
+except:
+    HAVE_DATALAD = False
+
+
+global local_testing_data_folder
+if os.getenv('EPHY_TESTING_DATA_FOLDER', default=None) is not None:
+    local_testing_data_folder = Path(os.getenv('EPHY_TESTING_DATA_FOLDER'))
+else:
+    # set in home
+    local_testing_data_folder = Path.home() / 'ephy_testing_data'
+
+
+def get_local_testing_data_folder():
+    global local_testing_data_folder
+    return local_testing_data_folder
+
+
+def download_dataset(repo=None, remote_path=None, local_folder=None):
+    """
+    Download a dataset with datalad client.
+
+    By default it download the "NeuralEnsemble/ephy_testing_data" on gin platform
+    which is used for neo testing.
+
+    Usage:
+
+        download_dataset(
+                repo='https://gin.g-node.org/NeuralEnsemble/ephy_testing_data',
+                remote_path='blackrock/blackrock_2_1',
+                local_folder='/home/myname/Documents/')
+
+    Parameters
+    ----------
+    repo: str
+        The url of the repo.
+        If None then 'https://gin.g-node.org/NeuralEnsemble/ephy_testing_data'
+        is used
+    remote_path: str of Path
+        The distant path to retrieve (file or folder)
+    local_folder: str or Path or None
+        The local folder where to download the data.
+
+    Returns:
+    --------
+    local_path:
+        The local path of the downloaded file or folder
+    """
+    assert HAVE_DATALAD, 'You need to install datalad'
+
+    if repo is None:
```

Comment:
wouldn't it be better to directly use this as default value for the `repo` parameter in the function definition? Or share this default url with the other places where it is used? See my other comment in this PR...

Reply:
I prefer None in params to make the code more easy to understand. I will try a global url somewhere.